eng/pipelines/eng-common-sync.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     steps:
     - template: ./templates/steps/sync-directory.yml
       parameters:
-        CommitMessage: Sync eng/common directory with azure-sdk-tools repository
+        CommitMessage: Sync eng/common directory with azure-sdk-tools repository for $(Build.BuildNumber)
```

Comment:
I suspect we want the PR number not the build number as the build number will change for every iteration of the tools PR.